### PR TITLE
Create compose file to upgrade to custom version and run tests

### PR DESCRIPTION
**Pull request by dahfjkg**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -33,13 +32,15 @@ repositories {
...
-    compileOnly([group: 'com.mendix', name: 'public-api', version: "$mendixPublicApiVersion"])
-    testCompileOnly("com.mendix:public-api:$mendixPublicApiVersion")
+    compileOnly([group: 'com.mendix', name: 'public-api', version: "(,$mxPublicApiVersion]"])
+    testCompileOnly("com.mendix:public-api:(,$mxPublicApiVersion]")
 
     testImplementation('com.mendix.util:junit-helper:1.0.0') {
         exclude group: 'com.mendix', module: 'public-api'
     }
 
+    implementation(fileTree(mxRuntimeBundles).include('*.jar'))
```
Comment (passalaqua) [inline build.gradle:42]: Why was this added? Was it needed somewhere?

Comment (dahfjkg) [reply build.gradle:42]: yep, dependencies would not resolve without that when running compose

**Review by passalaqua — APPROVED** [review]
Rated E for everyone.
